lib/matplotlib/axes/_base.py

```diff
@@ -1410,10 +1412,14 @@ def set_anchor(self, anchor, share=False):
 
     def get_data_ratio(self):
         """
-        Returns the aspect ratio of the raw data.
+        Returns
```

Comment:
In general, I would like docstrings to have a summary line. Not least because this is used in summaries such as
![grafik](https://user-images.githubusercontent.com/2836374/55820632-d6e2f500-5afb-11e9-85c6-e9a820527212.png).

Transforming to a returns section does only add the minor value of stating that it's a float.

Reply:
Agreed.  Restored the cases where the summary line indeed fits in a single line (for the longer cases, e.g. get_xaxis_text1_transform, I think the Returns section is more readable).